packages/w3up-client/src/capability/usage.js

```diff
@@ -31,6 +31,32 @@ export class UsageClient extends Base {
 
     return out.ok
   }
+
+  /**
+   * Record egress data for the customer and served resource.
+   *
+   * Required delegated capabilities:
+   * - `usage/record`
+   *
+   * @param {import('../types.js').SpaceDID} space
+   * @param {API.EgressData} egressData
+   * @param {object} [options]
+   * @param {string} [options.nonce]
+   */
+  async record(space, egressData, options) {
+    const out = await record(
+      { agent: this.agent },
+      { ...options, space, egressData }
+    )
+    /* c8 ignore next 5 */
```

Comment:
question: is it possible to add a test for the error case so we can avoid the c8 ignore here?

Reply:
Sounds good. I will add that.

Reply:
@travis I tried to mock the `agent.invokeAndExecute` to return an error using Sinon, but for some reason, it doesn't work. I guess that's why the other capability also had the `c8 ignore` for this part.

Reply:
ok - I'm ok skipping this in the interest of shipping if you think it's going to be a rabbithole!